weblate/static/vendor/multi.css

```diff
@@ -8,7 +8,8 @@
 .multi-wrapper .selected-wrapper {
     box-sizing: border-box;
     display: inline-block;
-    height: 200px;
+    height: 35vh;
+    min-height: 200px;
```

Comment:
This is a third-party file updated by `scripts/yarn-update` so any changes to it will be discarded.

Reply:
Oh! That's why it was in `vendor`. What package is that?

Reply:
Yes, everything in vendor is third-party code. It is `multi.js` package.